### PR TITLE
Fix re-apply repmgr in case newer package is installed

### PR DESCRIPTION
**Pull request by plirglo**

### CODE REVIEW

File: ansible/playbooks/roles/postgresql/tasks/extensions/replication/packages.yml [hunk 1]
```diff
@@ -1,14 +1,43 @@
...
+      package:
+        name: "{{ _packages[ansible_os_family] }}"
+        state: present
+      vars:
+        _packages:
+          Debian:
+            - "{{ repmgr.package_name.Debian }}={{ repmgr.version.Debian + '-*' }}"
+            - "repmgr-common={{ repmgr_common_target_version }}"
+      module_defaults:
+        yum: { lock_timeout: "{{ yum_lock_timeout }}" }
```
Comment (przemyslavic) [inline ansible/playbooks/roles/postgresql/tasks/extensions/replication/packages.yml]: yum defaults for Debian family? 😁

Comment (plirglo) [reply ansible/playbooks/roles/postgresql/tasks/extensions/replication/packages.yml]: Fixed

**Comment by przemyslavic** [comment]
:x:
`install 'repmgr-common=False'' failed: E: Version 'False' for 'repmgr-common' was not found`

It looks like the variable is resolved to False :cry:

```
2022-01-18T13:42:43.0812333Z1-1-1-1[38;21m13:42:43 INFO cli.engine.ansible.AnsibleCommand - TASK [postgresql : Extensions | repmgr | Install package(s) Debian] ************1-1-1-1
2022-01-18T13:42:43.9690970Z1-1-1-1[31;21m13:42:43 ERROR cli.engine.ansible.AnsibleCommand - fatal: [ci-10to13azubflannel-postgresql-vm-0]: FAILED! => {"cache_update_time": 1642512477, "cache_updated": false, "changed": false, "msg": "'/usr/bin/apt-get -y -o \"Dpkg::Options::=--force-confdef\" -o \"Dpkg::Options::=--force-confold\"      install 'repmgr-common=False'' failed: E: Version 'False' for 'repmgr-common' was not found\n", "rc": 100, "stderr": "E: Version 'False' for 'repmgr-common' was not found\n", "stderr_lines": ["E: Version 'False' for 'repmgr-common' was not found"], "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\n", "stdout_lines": ["Reading package lists...", "Building dependency tree...", "Reading state information..."]}1-1-1-1
2022-01-18T13:42:44.0949743Z1-1-1-1[31;21m13:42:44 ERROR cli.engine.ansible.AnsibleCommand - fatal: [ci-10to13azubflannel-postgresql-vm-1]: FAILED! => {"cache_update_time": 1642512477, "cache_updated": false, "changed": false, "msg": "'/usr/bin/apt-get -y -o \"Dpkg::Options::=--force-confdef\" -o \"Dpkg::Options::=--force-confold\"      install 'repmgr-common=False'' failed: E: Version 'False' for 'repmgr-common' was not found\n", "rc": 100, "stderr": "E: Version 'False' for 'repmgr-common' was not found\n", "stderr_lines": ["E: Version 'False' for 'repmgr-common' was not found"], "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\n", "stdout_lines": ["Reading package lists...", "Building dependency tree...", "Reading state information..."]}1-1-1-1

```

**Review by erzetpe — APPROVED** [review]

**Review by przemyslavic — APPROVED** [review]

**Comment by przemyslavic** [comment]
Please also create a PR to develop :)

**Pull request merged**